docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,384 @@
+  <source src="https://goteleport.com/teleport/videos/server-access/replay.webm" type="video/webm"/>
+Your browser does not support the video tag.
+</video>
+
+## Step 2b. Use tsh to introspect cluster
+
+1. Now, `auth-admin` has the ability to SSH into other Nodes within the cluster, traverse the Linux file system, and execute the command.
+
+   - They have visibility into all resources within the cluster due to their defined and assigned role.
+   - They can also quickly view any Node or grouping of Nodes that've been assigned a `label`.
+
+2. Executing `tctl nodes ls` displays:
+   
+   ```bash
+   Nodename         UUID                                 Address        Labels                                
+   ---------------- ------------------------------------ -------------- ------------------------------------- 
+   ip-172-31-30-155 8f1e2719-3161-400f-b664-3279cf046511                env=example,hostname=ip-172-31-30-155 
+   ip-172-31-17-124 e7719967-b6eb-4fc0-8e53-00fac6d896d4 127.0.0.1:3022 env=example,hostname=ip-172-31-17-124 
+   ```
+
+3. As a result, `auth-admin` can execute `tsh ssh root@env=example` since each Node has [label](../admin-guide.mdx#labeling-nodes-and-applications) by default. 
+
+   - Customized labels can be defined in your `teleport.yaml` configuration file or during Node creation.
+   - This is a convenient feature that allows for more advanced queries. 
+   - Suppose an IP address changes. An admin can quickly find the current Node with that label since it remains unchanged.
+
+4. `auth-admin` can also execute commands on all Nodes that share a label vastly simplifying repeated operations:
+
+   - `tsh ssh root@env=example ls` will execute the `ls` command on each Node displaying the contents of each to the screen.
+  
+   We'll review configuring and managing short-lived certificates for programmatic API access below.
+
+## Step 3a. Set up GitHub SSO
+
+<video autoplay loop muted playsinline>
+  <source src="https://goteleport.com/teleport/videos/server-access/githubsso.mp4" type="video/mp4"/>
+  <source src="https://goteleport.com/teleport/videos/server-access/githubsso.webm" type="video/webm"/>
+Your browser does not support the video tag.
+</video>
+
+Teleport Server Access also supports using SSO for authentication. Let's add a new user through GitHub SSO:
+
+1. Make sure the user you'll add to Teleport is a GitHub administrator within your GitHub organization.
+
+   i. As your GitHub administrator, go to your organization's **"Settings"** > **"Developer settings"** > **"OAuth Apps"** page.
+
+   ii. Add an OAuth App to your organization's page taking care to create a Client Secret.
+
+   ![OAuth App](../../img/server-access/oauth_app_config.png)
+
+2. Within your `auth` terminal:
+
+   - Execute `sudo nano /etc/teleport.yaml` and add the following entry to the `auth_service` section:
+
+   ```yaml
+   authentication:
+     type: "saml"
```

Comment:
`type: github`?

Reply:
Hmm... I think the current type is borrowed from the Okta SAML YAML config. I see there's a GitHub-specific format briefly mentioned in the Admin page.